docs/migrations.rst

```diff
@@ -30,4 +30,12 @@ General
   ``rasa.nlu.run`` -> ``rasa shell``
   ``rasa.nlu.server`` -> ``rasa run nlu``
   ``rasa.nlu.test`` -> ``rasa test nlu``
-  ``rasa.nlu.train`` -> ``rasa train nlu``
\ No newline at end of file
+  ``rasa.nlu.train`` -> ``rasa train nlu``
+
+
+Script parameters
+~~~~~~~~~~~~~~~~~
+- the ``--num_threads`` parameter got removed from the ``run`` command. The
```

Comment:
```suggestion
- the ``--num_threads`` parameter got removed from the ``server`` command. The
```

Reply:
I think, there is no `server` command. So, `run` should be fine.

Reply:
ah, I get you now 👍 I thought about the old `python -m rasa_nlu.server` command. I'd add this to the migrations as well.

Reply:
Should be already in there :)